DataProc/PPass/main_rec.C

```diff
@@ -129,6 +129,13 @@ void main_rec(const char *filename="raw.root", const char* options="")
   rec.SetDeleteRecPoints(delRecPoints.Data()); 
   //
 
+  const char* confPID = gSystem->Getenv("CONFIG_HE3_PION_THRESHOLD");
```

Comment:
Hi,
if I am not wrong, the environment variable that are set in the JDL (and I guess we will need to do it via JDL also for this special reco) have a prefix ALICE_JDL that is added automatically.
@catalinristea can confirm.
If this is the case, you should use ALICE_JDL_CONFIG_HE3_PION_THRESHOLD here.

Reply:
I gather from the other variables around that the prefix is `ALIEN_JDL_`, I adapted the new variable to this naming convention. But let's see what @catalinristea says! Thanks for noticing also this.

Reply:
Yes, I also confirm that you need the prefix if you want to read an env variable set in the JDL.

Reply:
Ciao,

I'm testing in grid to make sure it works. I'm not sure about the underscores in the name.
Thanks.

Yours,
Catalin.

Reply:
Thanks Catalin. I guess you have doubts in the underscores in the "CONFIG_HE3_PION_THRESHOLD" part, right? There I don't know if they are accepted or not. We can check with @costing ?

Reply:
Ciao,

I confirm it works. Please go on and merge, if nothing else.
Thanks!

Yours,
Catalin.

Reply:
Done, thanks!